Main_Data_classes_4.rb

```diff
@@ -0,0 +1,6 @@
+# 4. There are two strings. Find number of first matching chars
+
+def method_4 (str1, str2)
+  (str1.chars & str2.chars).length
```

Comment:
`&` this  method will find all common elements for both arrays but here we want to find only first matching elements.
For example:
str1 = '12lkjsdflkjsdf'
str2 = '12luhuhujhk'
we have only 3 first matching elements